apps/publisher/publisher.config.ts

```diff
@@ -0,0 +1,83 @@
+import { defineConfig } from '@amazeelabs/publisher';
+
+const isLagoon = !!process.env.LAGOON;
+
+const base = {
+  databaseUrl: '/tmp/publisher.sqlite',
+  publisherPort: isLagoon ? 3000 : 8000,
+  oAuth2: isLagoon
+    ? {
+        clientId: process.env.PUBLISHER_OAUTH2_CLIENT_ID || 'publisher',
+        clientSecret: process.env.PUBLISHER_OAUTH2_CLIENT_SECRET || 'publisher',
+        sessionSecret: process.env.PUBLISHER_OAUTH2_SESSION_SECRET || 'banana',
+        tokenHost:
+          process.env.PUBLISHER_OAUTH2_TOKEN_HOST || 'http://127.0.0.1:8888',
+        environmentType:
+          process.env.PUBLISHER_OAUTH2_ENVIRONMENT_TYPE || 'development',
+        scope: 'publisher',
+        tokenPath: '/oauth/token',
+        authorizePath: '/oauth/authorize?response_type=code',
+        grantType: 0,
+      }
+    : undefined,
+};
+
+export default defineConfig(
+  isLagoon
+    ? {
+        ...base,
+        mode: 'github-workflow',
+        publisherBaseUrl: `https://${process.env.SERVICE_NAME}.${process.env.LAGOON_ENVIRONMENT}.${process.env.LAGOON_PROJECT}.${process.env.LAGOON_KUBERNETES}`,
+        workflow: 'fe_build.yml',
+        repo: 'AmazeeLabs/silverback-template',
```

Comment:
This should be updated by `INIT.md`, or people will be very confused.

Reply:
Good catch, thank you!

Although, `pnpx @amazeelabs/mzx run INIT.md` is broken. It does nothing at all. I will fix it later. (I would actually replace it with a TS file and run via `tsx`)